src/Command/Command.php

```diff
@@ -89,61 +86,27 @@ public function destroyTmpDir()
         $this->tmpDir = "";
     }
 
-    protected function initialize(InputInterface $input, OutputInterface $output)
+    protected function initialize(InputInterface $input, OutputInterface $output): void
     {
-        if (!$this->checkRequirements($output)) {
-            throw new \RuntimeException("Requirements not satisfied - exiting.");
-        }
         // require sapi client
         $this->getSapiClient();
     }
 
-    /**
-     * @return FormatterHelper
-     */
-    public function getFormatterHelper()
+    public function getFormatterHelper(): FormatterHelper
     {
         return $this->getHelper('formatter');
     }
 
-    /**
-     * @return QuestionHelper
-     */
-    public function getQuestionHelper()
+    public function getQuestionHelper(): QuestionHelper
     {
         return $this->getHelper('question');
     }
 
-    /**
-     * @return NestedFormatterHelper
-     */
-    public function getNestedFormatterHelper()
+    public function getNestedFormatterHelper(): NestedFormatterHelper
     {
         return $this->getHelper('nestedFormatter');
     }
 
-    private function checkRequirements(OutputInterface $output)
```

Comment:
JFYI: minimálně curl a gzip by se daly přidat do composeru, ale v dockeru to potřeba není

```
{
    "require" : {
        "php" : "^5.5 || ^7.0",
        "ext-mbstring": "*"
    }
}
```